pgd_splicer/tests.py

```diff
@@ -329,3 +329,49 @@ def test_crosscheck_cullpdb(self):
 
         good_out = MonkeyPatch.sitefile('cullpdb_crosscheck.txt')
         self.assertEqual(test_out, file(good_out).read())
+
+
+class ProcessPDBTask(TestCase):
+
+    def test_import_pdb(self):
+
+        # Choose the first protein in the selection file.
+        with open(MonkeyPatch.sitefile('cullpdb_selection.txt'), 'r') as f:
+            line = f.readline().split(' ')
+
+        # Build pdbs variable.
+        # NB: structure taken from process_args() in ProcessPDBTask.main()
+        pdbs = [
```

Comment:
Should be a simple matter of moving the conversion between selection-file line and per-file data structure further down the chain.  It might be possible at another time to eliminate the conversion entirely if we can identify a single method that requires all the details, but that's a rabbit hole I want to avoid today.